pgbackrest_auto

```diff
@@ -590,6 +599,8 @@ touch "${log}"
 exec &> >(tee -a "${log}")
 pg_port_pick
 info "[STEP $((STEP++))]: Starting"
+# Reset values in zabbix status file before new restore
+printf "Restore_from_backup=0\nRestoring_from_archive=0\nData_validation=0\nPG_checksums_validation=0\nAmcheck_validation=0\nResult_status=1" > ${zbx_status_file}
```

Comment:
why `Result_status=1` ?

Reply:
because otherwise zabbix will show a recovery error alert until the script finishes its work.